gulpfile.js

```diff
@@ -82,6 +83,66 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  let path = require('path');
+  let rootDir = 'dist';
+
+  let config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      `${rootDir}/1.0/index.html`,
+      `${rootDir}/1.0/shell`,
```

Comment:
This shouldn't be listed in both `staticFileGlobs` and `dynamicUrlToDependencies`, I think.